### ISSUE TITLE
Add a way to stop navigation in the application

### ISSUE THREAD

**Issue by Deuchnord** [osmandapp/Osmand#5133]
Currently, the only way to stop navigation seems be by opening the notifications, find OsmAnd's one, expand it and tap the "Stop" button. Problem is, depending on the customizations provided by the interface of your phone, this button may be really difficult to reach - and can lead to a very dangerous behavior. For instance, on Huawei's EMUI (screenshot below), notifications have a very small buttons, so you have to be very precise to tap it.
It would be very interesting to have a button that would be as accessible as possible in the app - for instance, a floating button in the bottom right? - so we could stop navigation mode quickly while driving.

![OsmAnd's notification, as it appears in Huawei's default UI](https://user-images.githubusercontent.com/7600265/37556735-526a9ba8-29fa-11e8-86a1-360837b157cf.png)

**Comment by vshcherb** [comment]
No, there is another way to stop navigation, click on the button directions or menu in the drawer directions and then button X (cancel navigation)

**Comment by Deuchnord** [comment]
Oh, sorry, you're right, I didn't notice that buttons showed up when you touch the screen.
Thanks :)

**Comment by tremby** [comment]
I'm a new user today and could not find this. Even with those instructions it was incredibly hard to find.

Even after finding it that once, I started navigation again to see if I could find it again easily and no, I couldn't.

The "menu" and "navigation arrow" buttons are hidden until I tap the screen, but most of the time whatever I've tapped causes a "what's here" thing to open, and usually when I dismiss that by swiping it away the buttons I wanted are hidden again. I've found now that I need to slightly pan the map, and this way the buttons appear without a "what's here" dialog appearing. Then I can click either the menu hamburger then "navigation", or the navigation arrow, then "dismiss", then "yes".

It's not intuitive in the least.
